django/core/mail/backends/filebased.py

```diff
@@ -17,9 +17,6 @@ def __init__(self, *args, file_path=None, **kwargs):
             self.file_path = file_path
         else:
             self.file_path = getattr(settings, 'EMAIL_FILE_PATH', None)
-        # Make sure self.file_path is a string.
-        if not isinstance(self.file_path, str):
-            raise ImproperlyConfigured('Path for saving emails is invalid: %r' % self.file_path)
```

Comment:
With this reinstated, it looks as though a test is needed as you didn't remove one...